PANDORA/PMHC/PMHC.py

```diff
@@ -570,3 +584,21 @@ def fill_allele_seq_info(self, use_templ_seq=False):
             except:
                 print('\nWARNING: something went wrong when trying to retrieve chain M allele')
                 print('with blast. Is blastp properly installed as working as "/bin/bash blastp"?')
+
+    def make_output_dir(self):
+        ''' Create an output directory and move the template pdb there
+            Uses self.output_dir (str): Path to output directory. Defaults to os.getcwd().
+        Args:
+            None
+        '''
+
+        # create an output directory
+        try:
+            self.output_dir = '%s/%s' %(self.output_dir, self.id)
```

Comment:
Please do not redefine the output_dir here, do it directly in the __init__ function and don't change it later.

Reply:
If possible change this to f-string formatting also